_data/sites.json

```diff
@@ -10254,6 +10265,7 @@
         "notes": "Click on the Delete Account button and follow the steps.",
         "domains": [
             "protonvpn.com",
+            "proton.me",
```

Comment:
Adding the same domain on two different services seem weird, as it will make the browser extension confused on what to recommend.

Reply:
Ok, thanks por advice. The site was removed.